bitbake/lib/bb/fetch2/__init__.py

```diff
@@ -1398,7 +1398,7 @@ def unpack(self, urldata, rootdir, data):
 
         if unpack:
             if file.endswith('.tar'):
-                cmd = 'tar x --no-same-owner -f %s' % file
+                cmd = 'unpigz < %s | tar x --no-same-owner -f' % file
```

Comment:
Copy-paste bug (my bad): -f option is not needed
cmd = 'unpigz < %s | tar x --no-same-owner' % file